temporal/t.rast.colors/t.rast.colors.py

```diff
@@ -148,13 +148,10 @@ def main():
     if rows:
         # Create the r.colors input file
         filename = gs.tempfile(True)
-        file = open(filename, "w")
-
-        for row in rows:
-            string = "%s\n" % (row["id"])
-            file.write(string)
-
-        file.close()
+        with open(filename, "w") as file:
```

Comment:
maybe you can shorten it with `join`

Reply:
Or if it gets changed just a bit, then FURB122 would apply and suggest fixes like here: 

https://github.com/OSGeo/grass/pull/4982/commits/bd8d98c22b349bbe1ab7dd04e4d9329d3f7c6cdc